src/Microsoft.ML.Core/Data/AnnotationUtils.cs

```diff
@@ -440,7 +440,7 @@ public static IEnumerable<SchemaShape.Column> GetTrainerOutputAnnotation(bool is
         /// <param name="labelColumn">Label column.</param>
         public static IEnumerable<SchemaShape.Column> AnnotationsForMulticlassScoreColumn(SchemaShape.Column? labelColumn = null)
         {
-            var cols = new List<SchemaShape.Column>();
+           var cols = new List<SchemaShape.Column>();
```

Comment:
As far as I see the only change in this file is this incorrect indentation. Perhaps we want to fix this?